cart/cart.js

```diff
@@ -23,5 +23,5 @@ orderButton.addEventListener('click', () => {
     alert(cartAsAString);
 
     localStorage.clear();
-    window.location.href = '../';
+    window.location.href = '/';
```

Comment:
So it looks like you 'reverted' this branch and changed this path back to /. I think you had it right the last time, and if you change it, it will reroute to your home page on deployment